internal/file/structure.go

```diff
@@ -19,13 +19,13 @@ package file
 const (
 	// AuthDirectory default directory used to store files
 	// to authenticate request
-	AuthDirectory = "/etc/ingress-controller/auth"
+	AuthDirectory = "/tmp/ingress-controller/auth"
```

Comment:
Wouldn't it be enough to pre-create these directories with sufficient permissions instead?

Reply:
yes, it makes sense. Changed